kairos-test-utils/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "kairos-test-utils"
+version = "0.1.0"
+edition = "2021"
```

Comment:
I noticed that all packages (excluding prover) are inheriting from the workspace:

```
version.workspace = true
edition.workspace = true
```

Should it be the same case here?

Reply:
done https://github.com/cspr-rad/kairos/pull/34/commits/4276b0b3b7d0937e4755d040a8fc8c1e22562521